.gitignore

```diff
@@ -59,3 +60,7 @@ typings/
 
 # next.js build output
 .next
+
+#my conditions
+package-lock.json
```

Comment:
package-lock is useful file, don't add it in gitignore unless you are sure you want to do it

Reply:
done